### ISSUE TITLE
  convert onnx to  MNN  with pytorch " F.normalize  "

### ISSUE THREAD

**Issue by alicera** [alibaba/MNN#870]
平台
Ubuntu 16.04

Github版本:
1.0.1
pip安装

编译方式:
 
我在pytorch 加了簡單的
x = F.normalize(a, p=2, dim=1)
結果ONNX轉MNN時就發生以下錯誤

# 编译日志:
# Build Log:

```
MNNConverter Version: 0.2.1.5git - MNN @ 2018


Start to Convert Other Model Format To MNN Model...
[02:54:20] /MNN/tools/converter/source/onnx/onnxConverter.cpp:29: ONNX Model ir version: 4
terminate called after throwing an instance of 'Error'
  what():  [02:54:20] /MNN/tools/converter/source/onnx/Clip.cpp:37: Check failed: (maxValue) == (6.0f) ==> 
Aborted (core dumped)
```

**Comment by yyfcc17** [comment]
能提供一下可复现该错误的最小模型吗？

**Comment by alicera** [comment]
https://github.com/TreB1eN/InsightFace_Pytorch
https://drive.google.com/file/d/1JP-cNUaAKX9zTzUjj2yf6Cuhhv7F31Cd/view?usp=sharing

**Comment by yyfcc17** [comment]
你可以用pytorch自带的math算子组合出这个normalize，这些基础的math算子，MNN都是支持的

**Comment by jxt1234** [comment]
clip 转换的问题最新代码已经修正
